dashboard/lti_new.py

```diff
@@ -192,7 +192,7 @@ def check_if_instructor(roles, username, course_id):
 def course_user_roles(roles, username):
     user_membership_roles = set([role for role in roles if role.find(COURSE_MEMBERSHIP) == 0])
     if not user_membership_roles:
-        logger.info(f'User {username} does not have course membership roles, must be admin {roles}')
+        logger.error(f'User {username} does not have course membership roles, but has {roles} roles')
```

Comment:
Okay thanks. I don't think this need to be bumped up to an error. In many cases they may be an administrator. But I think the problem came up on the certification tests of 

However they said this would pass, it's just the message was confusing. 

* Launch Instructor with Short Role Value
``` 
[2022-05-05 15:26:24 -0400] [INFO] [lti_new.py:194] User l.c.hutton@district1.com does not have course membership roles, must be admin ['Instructor']
```

* Launch Instructor with Unknown Role
```
[2022-05-05 15:28:46 -0400] [INFO] [lti_new.py:194] User l.c.hutton@district1.com does not have course membership roles, must be admin ['http://purl.imsglobal.org/vocab/lis/v2/unknown/unknown#Helper' 
```

* Launch Instructor with No Role
```
[2022-05-05 15:31:46 -0400] [INFO] [lti_new.py:194] User l.c.hutton@district1.com does not have course membership roles, must be admin ['']
```

Reply:
I think our search for

`COURSE_MEMBERSHIP = 'http://purl.imsglobal.org/vocab/lis/v2/membership'

Was where this was failing. So it doesn't find course membership roles but there are some roles. I don't know if we need to do anything for the No Role, even though they say it's fine for certification.

Reply:
I changed the logic of how instructor **only** is checked. I log message only if he is an instructor. 

This check was in place since we only allow instructors to enable the data extraction process in MyLA after initial launch of the tool. Apart from that, we all allow MyLA Admin to enable the data extraction process ( or to say have a course entry in `course` table. 

3 users we need to test with 

1. Admins in Canvas
2. Instructor in a course
3. Instructor and TA in a course
4. TA only
5. Admin In MyLA